### ISSUE TITLE
ValueError: all the input array dimensions for the concatenation axis must match exactly, but along dimension 1, the array at index 0 has size 4 and the array at index 5 has size 3

### ISSUE THREAD

**Issue by IoanaPreda** [YttriLab/A-SOID#91]
**Describe the bug**
At the Active Learning step, whilst being able to view the performance of the classifier with ~60 iterations, upon refreshing the page an error is encountered. Additionally, no iterations can be accessed (except iteration 0) after completing this step.

**To Reproduce**
1. Go to 'Active Learning'
2. Click on 'Train Classifier'
3. Reload the page after viewing the performance plot OR click on 'Show final results'

**Expected behavior**
The performance plot was expected to be seen whenever going back to the 'Active Learning' step together with an additional plot of the iterations of active learning.  It was also expected to have the option to select a specific iteration for the following steps, but none are available.

**Screenshots**
Error:
![image](https://github.com/YttriLab/A-SOID/assets/139000444/dc63be1d-bd9b-45f8-806c-6b1a763aae21)

Active learning results:
![image](https://github.com/YttriLab/A-SOID/assets/139000444/fdf2cff6-95a0-4cb9-a713-26d29ae7e685)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 126.0.6478.114 

**Project Config (please post the content of the corresponding config.ini file)**
![image](https://github.com/YttriLab/A-SOID/assets/139000444/17a9083c-a193-4140-91e4-3bc447f2acfe)

**Additional context**
This was the latest attempt at training the classifier and running into the same error. Possible solutions tried: starting a new session, a new project and a new environment.

**Comment by JensBlack** [comment]
Hi @IoanaPreda ,

Thank you for reporting that issue. It might be the same as was reported at #79.
Which version of A-SOiD did you install? This bug should have been fixed a while ago.

> This bug is related to a plot (see other issue) and should not affect the underlying functions, you can continue working on the other steps while I am hunting it down.

Iteration 0 is the iteration you get when training active learning once. Every other iteration is only happening after refinement steps. 

Did you do refinement in this project?

**Comment by IoanaPreda** [comment]
Hi Jens,

Thank you for your reply.

The version I have is: "0.3.1.2", which I believe includes the latest updates.

Thanks for clarifying that point. Unfortunately, I am unable to do the refinement step due to the 2GB video file requirement. Can this part be modified for larger files (~9GB)?

**Comment by JensBlack** [comment]
Yes, the file size can be altered in the code, but refinement is an optional step (and often not needed if you feed in enough data in the first place). You can create a new project with additional data/labels and see whether it improves your performance.

Yes, this should be included in the recent update... I will take a look at it.

**Comment by JensBlack** [comment]
@IoanaPreda thank you for your patience. I was finally able to fix the plotting issue.
